gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -188,22 +188,39 @@ def run_assertion(self):
 class QueryFormatter(object):
   """Formats a query.
 
-  Replaces keyword TABLE_NAME and eventually macros in the query.
+  Replaces macros and variable TABLE_NAME in the query.
   """
 
+  class _QueryMacros(enum.Enum):
+    NUM_ROWS_QUERY = 'SELECT COUNT(0) AS num_rows FROM {TABLE_NAME}'
+    SUM_START_QUERY = ('SELECT SUM(start_position) AS sum_start FROM {'
+                       'TABLE_NAME}')
+    SUM_END_QUERY = 'SELECT SUM(end_position) AS sum_end FROM {TABLE_NAME}'
+
   def __init__(self, table_name):
     # type: (str) -> None
     self._table_name = table_name
 
   def format_query(self, query):
-    # type: (List[str]) -> str
+    # type: (list[str]) -> str
     """Formats the given ``query``.
 
     Formatting logic is as follows:
     - Concatenates ``query`` parts into one string.
+    - Replaces macro NUM_ROWS_QUERY/SUM_START_QUERY/SUM_END_QUERY with the
```

Comment:
Consider not listing the members of _QueryMacros here such that adding a new query or changing the name of one in _QueryMacros does not an update of this documentation (which will probably not happen making the documentation obsolete in future).

Reply:
Done.